admin/src/Membership/MemberBoxMemberData.jsx

```diff
@@ -2,21 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Member from "../Models/Member";
 import MemberForm from "../Components/MemberForm";
-import {confirmModal} from "../message";
-import {withRouter} from "react-router";
+import { confirmModal } from "../message";
+import { withRouter } from "react-router";
+import auth from '../auth';
 
 
 class MemberBoxMemberData extends React.Component {
+
+    update_member_info(member_number) {
```

Comment:
The API always works with member ids, not member numbers, except in some rare special cases.

Reply:
Ok, I've modified that now! :)